### DISCUSSION TITLE
With PWA configuration on next js next/image hostname configuration not working

### DISCUSSION THREAD

**Discussion by avabhi** [vercel/next.js#44079] [Help]
I have PWA app with next js projects ,in which I want to use next/image for image optimisation But it's giving me error "Error: Invalid src prop [(https://picsum.photos/id/0/120/120)](http://localhost:5002/tempApp/(https://picsum.photos/id/0/120/120)) on `next/image`, hostname "picsum.photos" is not configured under images in your `next.config.js`"

Even after restarting server multiple times, it's not working.

Below is my configuration for domains in next.config.js

![Screenshot 2022-12-16 at 7 04 27 PM](https://user-images.githubusercontent.com/32228794/208109983-a3becd94-6b4d-4c16-81a6-b70a06c44c39.png)
![Screenshot 2022-12-16 at 7 05 14 PM](https://user-images.githubusercontent.com/32228794/208109991-13b63978-a880-497d-a7e5-ead5a2e7ddc5.png)
![Screenshot 2022-12-16 at 7 05 53 PM](https://user-images.githubusercontent.com/32228794/208109997-bd467ccd-5d74-4be5-b43c-ffb8311f13aa.png)

**Comment by icyJoseph** [comment]
If you were to remove, withPWA or withOffline from the `next.config.js` does it work then? I suspect one those two is not passing along the images key.


## Edit

It looks like you are using `withPWA` incorrectly:

```js
const withPWA = require('next-pwa')({
  dest: 'public'
})

module.exports = withPWA({
  // next.js config
})
```

Notice that you ought to call the required module! The API for `next-pwa` changed recently, so in your case it'd be (since you posted an image I won't copy all the configs):

```js
const withOffline = require("next-offline")
const withPWA = require('next-pwa')({
  dest: 'public',
  register: true,
 // ...rest of stuff
})

// this next line is misplaced on your example
/** @type {import('next').NextConfig} */
const nextConfig = {
}

module.exports = withPWA(withOffline(nextConfig))
```
